testing/trino-testing-services/src/main/java/io/trino/testng/services/ProgressLoggingListener.java

```diff
@@ -137,22 +137,7 @@ public void onAfterClass(ITestClass testClass)
             return;
         }
 
-        LOGGER.info("[AFTER CLASS] %s", getName(testClass));
-    }
-
-    private String formatTestName(ITestResult testCase)
```

Comment:
Were these two methods unused?

Reply:
They are used - i moved them to `Listeners`